### PR TITLE
Added 'shepherd-popper' css class

### PR DESCRIPTION
**Pull request by hope55**
Added 'shepherd-popper' css class to the top level 'div.tippy-popper' element, in order to isolate it's shepherd specific styles from other tippy styles on the page. (Changed files outside of `src` folder can be ignored.) I run manual tests and `yarn test:cy:watch`. All tests have passed.

### CODE REVIEW

File: src/js/utils.js [hunk 1]
```diff
@@ -5,6 +5,15 @@ import zipObject from 'lodash-es/zipObject';
 import tippy from 'tippy.js';
 import { missingTippy } from './utils/error-messages';
 
+// popperOption modifier, to add shepherd-popper class to both default and centeredStyle poppers
+const addShepherdClass = {
+  enabled: true,
+  fn: (data) => {
+    data.instance.popper.classList.add('shepherd-popper');
```
Comment (rwwagner90) [inline src/js/utils.js]: Does this class need to be added to popper or tippy?

**Comment by rwwagner90** [comment]
@hope55 I went ahead and approved the PR, but there is a conflict. Can you rebase and fix the conflict please? Sorry about that!

**Comment by hope55** [comment]
@rwwagner90 I'm not sure if I have anything left to do. As I see you could merge the PR. Did you manage to resolve the conflict? I haven't seen any conflicts or failed tests after my latest commit (cf2e4b2).

**Comment by rwwagner90** [comment]
@hope55 there was a conflict, so I opened a new PR and merged it. Everything should be in now. Thanks so much!

**Pull request closed without merging**